### PR TITLE
updated redmi

### PR DESCRIPTION
**Pull request by badesz97**
updated sziami redmi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+-   The development should include <b>tests</b>.
+-   The microservices should communicate through <b>REST</b> interfaces.
+-   Messaging platforms like <b>Kafka</b> or <b>NATS</b> should be used.
+-   Databases should be either <b>document oriented</b> or <b>SQL</b>.
+-   Frontend frameworks such as <b>Angular</b> or <b>React</b> should be used.
+
 ## Structure
-  - Frontend
-  - Backend
+
```
Comment (szerg97) [inline README.md]: delete this line

File: README.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
+-   Frontend
+-   Backend
+-   Midend :)
+
```
Comment (szerg97) [inline README.md]: delete this line

File: README.md [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
-  - Databases should be either <b>document oriented</b> or <b>SQL</b>.
-  - Frontend frameworks such as <b>Angular</b> or <b>React</b> should be used.
+
+-   The goal is to combine different microservices (written in JavaScript and Java) to create backend and later frontend.
+-   The development should include <b>tests</b>.
+-   The microservices should communicate through <b>REST</b> interfaces.
+-   Messaging platforms like <b>Kafka</b> or <b>NATS</b> should be used.
+-   Databases should be either <b>document oriented</b> or <b>SQL</b>.
+-   Frontend frameworks such as <b>Angular</b> or <b>React</b> should be used.
+
```
Comment (szerg97) [inline README.md]: delete this line

File: README.md [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
 Booking application with microservices including <b>TypeScript</b> and <b>Spring Boot</b>.
+
 ## Goals
-  - The goal is to combine different microservices (written in JavaScript and Java) to create backend and later frontend.
-  - The development should include <b>tests</b>.
-  - The microservices should communicate through <b>REST</b> interfaces.
-  - Messaging platforms like <b>Kafka</b> or <b>NATS</b> should be used.
-  - Databases should be either <b>document oriented</b> or <b>SQL</b>.
-  - Frontend frameworks such as <b>Angular</b> or <b>React</b> should be used.
+
```
Comment (szerg97) [inline README.md]: delete this line

File: README.md [hunk 5]
```diff
@@ -1,13 +1,20 @@
 # Booking application
+
 Booking application with microservices including <b>TypeScript</b> and <b>Spring Boot</b>.
+
```
Comment (szerg97) [inline README.md]: delete this line

File: README.md [hunk 6]
```diff
@@ -1,13 +1,20 @@
 # Booking application
+
```
Comment (szerg97) [inline README.md]: delete this line

**Review by szerg97 — CHANGES_REQUESTED** [review]
Fix the lines

File: README.md (hunk 3 above)
Comment (badesz97) [reply README.md]: szopki nemcsinalommeg

**Review by szerg97 — APPROVED** [review]
Nice job kid :)

**Pull request merged**